bom/pom.xml

```diff
@@ -103,11 +105,12 @@
                 <groupId>org.hibernate</groupId>
                 <artifactId>hibernate-envers</artifactId>
                 <version>${hibernateVersion}</version>
-            </dependency>
```

Comment:
The idea of the BOM is to give a user matching versions of all relevant
libs. Say you have an app using ORM@RDBMS + OGM and want to use Envers with
the former. Without that entry you'd have to match the Envers version with
the other ORM components defined by the BOM.